test/capybara_screenshot_diff/dsl_test.rb

```
@@ -0,0 +1,135 @@
+      assert_image_not_changed(["my_test.rb:42"], "name", comparison)
+      assert_not comparison.base_image_path.exist?
+    end
+
+    private
+
+    def our_screenshot(name, skip_stack_frames)
+      screenshot(name, skip_stack_frames: skip_stack_frames)
+    end
+
+    def assert_image_not_changed(*)
+      CapybaraScreenshotDiff::ScreenshotAssertion.assert_image_not_changed(*)
```

RuboCop (1.75.5)

[fatal] unexpected token tRPAREN
(Using Ruby 3.1 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)